explorer/explorerroutes.go

```diff
@@ -1631,6 +1631,24 @@ func (exp *explorerUI) AgendaPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	yes, abstain, no, err := exp.explorerSource.AgendaCummulativeVoteChoices(agendaId)
+	if err != nil {
+		log.Errorf("fetching cummulative votes choices count failed: %v", err)
+	}
+
+	// Overrides the default count entry which is always zero especially when the
```

Comment:
"always" and "especially" are a confusing combination.  Can you expand on what exactly agendaInfo has set at this point (from GetAgendaInfo).

Also, please rename all `GetAgendaInfo` to just `AgendaInfo`.  We're trying to stop using a "get" prefix unless there's a good reason.

Reply:
Noted.